tests/util/test_blockstore.py

```diff
@@ -0,0 +1,44 @@
+import hashlib
+import pathlib
+
+import pytest
+
+from inbox.util import blockstore
+
+
+@pytest.mark.usefixtures("blockstore_backend")
+@pytest.mark.parametrize("blockstore_backend", ["disk", "s3"], indirect=True)
```

Comment:
sync-engine has two blockstore backends, S3 and disk. We use S3 in production but disk in our local devtools setup.